massa-execution/src/types.rs

```diff
@@ -280,10 +287,7 @@ pub(crate) struct ExecutionContext {
     pub opt_block_creator_addr: Option<Address>,
 
     /// address call stack, most recent is at the back
-    pub call_stack: VecDeque<Address>,
-
-    /// list of addresses created so far during excution, stacked
-    pub owned_addresses_stack: Vec<Vec<Address>>,
+    pub stack: VecDeque<StackElement>,
```

Comment:
why a vecdeque ? a vec would make more sense and avoid many conversions below

Reply:
It also transforms a lot of calls to back into calls to last.

Reply:
I don't understand your argument. back() behaves just like last(), no difference and no performance loss.

Vec is especially designed to make stacks efficiently: https://doc.rust-lang.org/std/collections/index.html#use-a-vec-when